meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "surface-apps" %}
+{% set version = "0.1.0a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  path: ../{{ name }}
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
> The --no-deps and --no-build-isolation options ensure that pip does not itself install any dependencies. Since the conda environment is designed to already have the complete set of dependencies, the pip installation should succeed.